src/ui/public/index_patterns/index_patterns.js

```diff
@@ -41,6 +41,7 @@ export function IndexPatternsProvider(Notifier, Private) {
   self.cache = patternCache;
   self.getIds = getProvider('id');
   self.getTitles = getProvider('attributes.title');
+  self.getIndexPatterns = getProvider();
```

Comment:
Thanks for all the comments. I have a clear path forward. I will remove `_get.js` and move the getProvider functionality into index_patterns as @spalger suggested. I will name the new method `getSavedIndexPatterns ` as @Stacey-Gammon suggested.